SocialFilter/sections/specific.txt

```diff
@@ -6328,7 +6328,7 @@ animal.mx###ax_single_bar_sharers
 allcalidad.la###barraPlus1
 tecmundo.com.br##.tec--article__share-grid
 tecmundo.com.br##.tec--share
-artstation.com##.share-grid > .s-grid
+artstation.com##.share-grid
```

Comment:
@admitrevskiy Close the gate!)
Please revert, or change to 
`artstation.com##body .share-grid`